bindings/python/iota_sdk/types/signature.py

```diff
@@ -29,7 +29,7 @@ class Ed25519Signature(Signature):
     type: int = field(default_factory=lambda: 0, init=False)
 
 
-SignatureUnion: TypeAlias = Union[Ed25519Signature]
+Signature: TypeAlias = Ed25519Signature
```

Comment:
I don't know Python enough, wouldn't this be breaking change when we add more if it's not already a Union?

Reply:
No. And in fact my linter complained about using a union for a single variant

Reply:
Alright